src/richie/apps/courses/views.py

```diff
@@ -0,0 +1,27 @@
+from dal import autocomplete
+
+from cms.api import Page
+
+
+class PersonPageAutocomplete(autocomplete.Select2QuerySetView):
```

Comment:
We need to differentiate it from the Elasticsearch autocomplete for clarity:
```suggestion
class PersonPageAdminAutocomplete(autocomplete.Select2QuerySetView):
```